.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Release to latest/edge
+
+on:
+  # TODO: figure out a release process for the charming-actions
+  workflow_dispatch:
+  # push:
+  #   branches:
+  #     - main
+  #   paths-ignore:
+  #     - ".github/**"
+  #     - ".gitignore"
+  #     - "tox.ini"
```

Comment:
Why is this disabled? Because of the missing charmhub token?

Reply:
yes that and we would need to think about the release process first. So for now I have kept this only as a manual tirgger and removed the branch push hook

Reply:
I think then lets drop what we currently don't use from this PR an only get the iminimal things in.